stable/kommander/templates/grafana/hooks-home-dashboard.yaml

```diff
@@ -0,0 +1,80 @@
+{{- if .Values.grafana.enabled }}
+---
+# Unable to get post-install job hook working, which is why
+# this is a regular Job. The retries in the configmap script
+# should ensure that this successfully runs once the Grafana
+# server is up.
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Values.grafana.hooks.jobName | quote }}
+  namespace: {{ .Release.Namespace }}
+  labels:
```

Comment:
Should we add a hook delete policy for `hook-succeeded`? No need to keep the job and the pods around if everything worked

Reply:
Unfortunately I was not able to get the hook working, so it's just a regular job

Reply:
oh i see, i thought it was just the post-install hook but it was all hooks. Okay, all good.

Reply:
Please put a jira ticket to review this. we should be cleaning up after jobs.

Reply:
https://jira.d2iq.com/browse/D2IQ-63670